lib/iris/tests/test_pandas.py

```diff
@@ -236,8 +244,12 @@ def test_time_360(self):
         time_coord = DimCoord([100.1, 200.2], long_name="time",
                               units=time_unit)
         cube.add_dim_coord(time_coord, 0)
-        expected_index = [netcdftime.datetime(2000, 4, 11, 2, 24),
-                          netcdftime.datetime(2000, 7, 21, 4, 48)]
+        if netCDF4.__version__ > '1.2.4':
```

Comment:
I feel pretty bad about this approach, but it represents another step in a long line of trying to work with netcdf4python

i think that this 

there is also a particular issue with conda-forge builds of netcdf, not all versions are available as proper builds

further work is required on this, which needs to be planned
#2322